packages/store-sync/src/common.ts

```diff
@@ -42,9 +50,9 @@ export type SyncOptions<TConfig extends StoreConfig = StoreConfig> = {
    */
   address?: Address;
   /**
-   * Optional table IDs to filter indexer state and RPC state.
+   * Optional filters for indexer and RPC state. Useful to narrow down the data received by the client for large worlds.
    */
-  tableIds?: Hex[];
```

Comment:
should we still support `tableIds` as a shorthand?

if so, how do we support both `tableIds` and `filters`? combine them?